### DISCUSSION TITLE
What does priority mean on things other than rules?

### DISCUSSION THREAD

**Discussion by schicks** [CacheControl/json-rules-engine#365] [Q&A]
It looks from the type definitions as though facts and conditions both accept a `priority` field. In the context of a rule, priority seems to determine the order in which events will be emitted. What does it do on a fact or a condition?

**Comment by bodzilla** [comment]
Priority determines the order on which the conditions are assessed - the higher the number, the higher the priority:
Priority 1 = event will get triggered second
Priority 2 = event will get triggered first
